src/lib/rest/rest.cpp

```diff
@@ -871,6 +871,11 @@ static std::string apiVersionGet(const char* path)
     return "v2";
   }
 
+  if (strcmp(path, "/admin/log") == 0)
+  {
+    return "v2";
```

Comment:
The thing is, we use ciP->apiVersion to format OrionError and possible more responses.
Unless we are ok with errors like 

    {
        "orionError" : {
            ""
        }
    }

we need to 'fool ourselves' that the request is a v2 request ...